src/test/java/dev/devature/penguin_api/service/RegisterServiceTest.java

```diff
@@ -1,57 +1,54 @@
 package dev.devature.penguin_api.service;
 
 import dev.devature.penguin_api.entity.User;
+import dev.devature.penguin_api.utils.ValidationUtils;
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.mockito.InjectMocks;
-import org.mockito.Mock;
 import org.mockito.MockitoAnnotations;
 
 import java.util.Arrays;
 
 public class RegisterServiceTest {
 
-    @Mock
-    private User user;
-
     @InjectMocks
     private RegisterService registerService;
 
     @BeforeEach
-    public void setup() throws InterruptedException {
-
+    public void setup() {
         MockitoAnnotations.openMocks(this);
     }
 
     @Test
     public void emailRequirementTest(){
         String[] email = {"normal@example.com", "first.last@example.com", "user+tag@example.com",
-        "\"quoted@local\"@example.com", "special!#$%&'*+-/=?^_`{|}~@example.com", "plainaddress@.com",
-        "user@com", "user..name@example.com", "user@-example.com", "user@exa(mple).com"};
+                "\"quoted@local\"@example.com", "special!#$%&'*+-/=?^_{|}~@example.com", "plainaddress@.com",
+                "user@com", "user..name@example.com", "user@-example.com", "user@exa(mple).com"};
         boolean[] expectedResult = {true, true, true, true, true, false, false, false, false, false};
         boolean[] actualResult = new boolean[10];
 
-        for(int i = 0; i < email.length; ++i){
-            actualResult[i] = registerService.checkEmailRequirements(email[i]);
+        // Check email validity using the utility class
+        for (int i = 0; i < email.length; ++i) {
+            actualResult[i] = ValidationUtils.isValidEmail(email[i]);
         }
 
+        // Assert the results
         Assertions.assertArrayEquals(expectedResult, actualResult,
                 "Expected Boolean True, True, True, True, True, False, False, False, False, False - " +
                         "Actual Boolean was: " + Arrays.toString(actualResult));
     }
 
-    
     @Test
     public void passwordRequirementTest() {
         // Given
         String[] password = {"1234", "Test_Password12", "TestPassword12", "Test_Password"};
         boolean[] expectedResult = {false, true, false, false};
         boolean[] actualResult = new boolean[4];
 
-        // When
-        for(int i = 0; i < password.length; ++i){
-            actualResult[i] = registerService.checkPasswordRequirements(password[i]);
+        // Check password validity using the utility class
+        for (int i = 0; i < password.length; ++i) {
+            actualResult[i] = ValidationUtils.isValidPassword(password[i]);
```

Comment:
With the refactor moving those methods to `ValidationUtils`, these are really tests of that class, not `RegisterService` so could you make a test class for that and move these tests there